vrecord

```diff
@@ -1338,14 +1338,14 @@ if [[ "${FRAMEMD5_CHOICE}" = "Yes" ]] ; then
         _writeingestlog "PTS_DISCONTINUITY" "none"
     else
         _writeingestlog "PTS_DISCONTINUITY" "${PTS_DISCONTINUITY}"
-        cowsay "$(_report -w "WARNING: There were pts discontinuities for these frame ranges: ${PTS_DISCONTINUITY}. The file may have sync issues.")"
+        cowsay "$(_report -w "WARNING: There were discontinuities in the file's frame MD5s for these frame ranges: ${PTS_DISCONTINUITY}. This error may indicate frames dropped by FFmpeg or vrecord. The file may have sync issues.")"
```

Comment:
I think removing the reference to pts here changes the meaning. The discontinuities are not in the md5s but in the temporal differences between the sequential pts from the frame md5 report. You could say something like "There were presentation timestamp discontinuities found in the framemd5 for ..."

Reply:
Good point!